src/problem_depot/problem_depot.jl

```diff
@@ -105,12 +105,12 @@ MathOptInterface model, but not for the actual problem data.
 
 ```julia
 run_tests(exclude=[r"mip"]) do p
-    solve!(p, SCSSolver(verbose=0))
+    solve!(p, () -> SCS.Optimizer(verbose=0))
```

Comment:
I would write these like
```suggestion
    solve!(p, SCS.Optimizer; silent_solver=true)
```
these days, since then it's easier to swap between solvers. This uses the `MOI.Silent()` attribute instead of solver-specific ones.